src/test/java/com/solvd/movie/service/impl/EsMovieServiceTest.java

```diff
@@ -16,18 +16,16 @@
 @ExtendWith(MockitoExtension.class)
 public class EsMovieServiceTest {
 
-    private final FakeMovieService fakeMovieService = new FakeMovieService();
-
     @Mock
-    private EsRepository esRepository;
+    private EsMovieRepository esMovieRepository;
 
     @InjectMocks
     private EsMovieServiceImpl esMovieService;
 
     @Test
     public void retrieveAll() {
-        EsMovie movie = this.fakeMovieService.getEsMovie();
-        Mockito.when(this.esRepository.findAll())
+        EsMovie movie = FakeMovieService.getEsMovie();
+        Mockito.when(this.esMovieRepository.findAll())
```

Comment:
return here your movie, that you return in `FakeMovieService#getEsMovie()`

Reply:
This is the next string `.thenReturn(Flux.just(movie))`, I returned this object, or what do you mean?